src/components/ebay-combobox/index.marko

```diff
@@ -69,6 +69,7 @@ $ var id = input.id || component.getElId('input');
             aria-roledescription=roledescription
             aria-haspopup='listbox'
             autocomplete='off'
+            aria-expanded='false'
```

Comment:
If we render this with open as true or expanded, does it properly expand and set to true? I assume yes, but we should verify.

Reply:
I guess it should be. How can we test this? When I pass `&args=expanded:true` to [url](https://opensource.ebay.com/ebayui-core/?path=/story/form-input-ebay-combobox--floating-label&args=expanded:true), it doesn't load expanded on storybook.

Reply:
We need just a test case. Although it probably shouldn't be an issue since it will open client side.